detection-rules/impersonation_sharepoint_fake_file_share.yml

```diff
@@ -183,6 +186,15 @@ source: |
         regex.icontains(body.html.raw, 'rgb\(([0-4]),\s?(18[0-1]),\s?(24[0])\)'),
         regex.icontains(body.html.raw, 'rgb\((25[0-5]),\s?(20[0-2]),\s?([0-7])\)')
       )
+      or (
```

Comment:
At first this new logic gave me concern because it would match on messages that were sent from the org_domain which might actually contain the recipient domain as well.  But there is already logic to negate where the sender domain is in the $org_domains.